### PR TITLE
Removes a redundant var from a few machines

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Some consoles required a var edited dir to find the linked machine. This removes that need and just uses a range check.

This requires map edits, but they won't conflict as they are edits within keys only.

## Why It's Good For The Game
Less finicky vars, you can just plop the console one tile away from the machine and it works.

## Testing
Loaded game, saw consoles.

### CODE REVIEW

**Comment by SteelSlayer** [comment]
The amount of lines modified seems excessive, might need to merge master in this (whenever the map is unblocked anyway).

File: code/modules/mining/machine_processing.dm [hunk 1]
```diff
@@ -19,16 +19,17 @@
...
+	for(var/obj/machinery/mineral/processing_unit/found_machine in range(1, src))
+		machine = found_machine
+		break
+
 	if(machine)
-		machine.CONSOLE = src
-	else
-		return INITIALIZE_HINT_QDEL
+		machine.console = src
+		return
```
Comment (lewcc) [inline code/modules/mining/machine_processing.dm:32]:
```suggestion
```

File: code/modules/mining/machine_processing.dm [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
-	var/machinedir = EAST
 	speed_process = TRUE
 
 /obj/machinery/mineral/processing_unit_console/Initialize(mapload)
 	. = ..()
-	machine = locate(/obj/machinery/mineral/processing_unit, get_step(src, machinedir))
+	for(var/obj/machinery/mineral/processing_unit/found_machine in range(1, src))
+		machine = found_machine
+		break
+
```
Comment (warriorstar-orion) [inline code/modules/mining/machine_processing.dm:29]: why not 

```suggestion
	for(var/obj/machinery/mineral/processing_unit/found_machine in range(1, src))
		machine = found_machine
		machine.console = src
		return

```

File: code/modules/mining/machine_stacking.dm [hunk 3]
```diff
@@ -8,15 +8,18 @@
...
+		machine = found_machine
+		break
+
 	if(machine)
 		machine.console = src
-	else
-		return INITIALIZE_HINT_QDEL
+		return
+
+	return INITIALIZE_HINT_QDEL
```
Comment (hal9000PR) [inline code/modules/mining/machine_stacking.dm:22]: I feel this should throw a runtime when it fails to connect.

Apply warriors changes here as well btw.

File: code/modules/mining/machine_stacking.dm [hunk 4]
```diff
@@ -13,14 +13,9 @@
 	. = ..()
 	for(var/obj/machinery/mineral/stacking_machine/found_machine in range(1, src))
 		machine = found_machine
-		break
-
-	if(machine)
 		machine.console = src
 		return
 
-	return INITIALIZE_HINT_QDEL
```
Comment (hal9000PR) [inline code/modules/mining/machine_stacking.dm]: Doesn't it still need this incase it doesnt link to the other machine?

**Review by warriorstar-orion — COMMENTED** [review]
LGTM

**Review by hal9000PR — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**